src/test/java/teammates/test/cases/action/AdminActivityLogPageActionTest.java

```diff
@@ -379,7 +379,7 @@ public void removeAndRestoreManyLogs() {
                 today.getTime(), LOG_MESSAGE_INTERVAL_MANY_LOGS);
     }
 
-    @Test(groups = "manyLogs", dependsOnGroups = "typicalLogs")
+    @Test(groups = "manyLogs", priority = 2)
```

Comment:
I didn't mean that. Rather, the fact that this needs to be `>1` to avoid {whatever you are trying to avoid}.